.gitignore

```diff
@@ -137,7 +137,6 @@ token
 
 # E2E test files
 test-results
-e2e
```

Comment:
I didn't realise this was messing up all the test changes I have been doing for weeks locally :D